### ISSUE TITLE
Darknet19 pretrain weight transfer learning

### ISSUE THREAD

**Issue by visionNoob** [DeepBaksuVision/YOLO9000#15]
1. Darknet19 weight 가져오기 
2. Post hoc transfer learning에 대해서 데모

**Comment by visionNoob** [comment]
darknet cfg 가져올때 쓸만한
https://github.com/marvis/pytorch-caffe-darknet-convert/blob/master/cfg.py

**Comment by visionNoob** [comment]
1: MaxPool2d
    2: Sequential - conv2d, bn2, leaky2 64
    3: MaxPool2d
    4: Sequential - conv3d, bn3, leaky3 128
    5: Sequential - conv4d, bn4, leaky4 64
    6: Sequential - conv5d, bn5, leaky5 128
    7: MaxPool2d
    8: Sequential - conv6d, bn6, leaky6 256
    9: Sequential - conv7d, bn7, leaky7 128
    10: Sequential - conv8d, bn8, leaky8 256
    11: MaxPool2d
    12: Sequential - conv9d, bn9, leaky9 512
    13: Sequential - conv10d, bn10, leaky10 256
    14: Sequential - conv11d, bn11, leaky11 512
    15: Sequential - conv12d, bn12, leaky12 256
    16: Sequential - conv13d, bn13, leaky13 512
    17: MaxPool2d
    18: Sequential - conv14d, bn14, leaky14 1024
    19: Sequential - conv15d, bn15, leaky15 512
    20: Sequential - conv16d, bn16, leaky16 1024
    21: Sequential - conv17d, bn17, leaky17 512
    22: Sequential - conv18d, bn18, leaky18 1024
    23: Sequential - conv19d 1000
    24: GlobalAvgPool2d
    25: Softmax())
